### PR TITLE
Feature/add bootstrap

### PR DESCRIPTION
**Pull request by monica-nathalia**
add external .css for all to refer to. ignore the database password (local to my computer only).

Also, is the 'settings' file for everyone has changed to 'settings_template'?

### CODE REVIEW

File: app/templates/base.html [hunk 1]
```diff
@@ -1,15 +1,29 @@
...
+{% load staticfiles %}
+<html lang="en">
   <head>
     <meta charset="utf-8">
-    <title>IMSLPBAR/not Petrucci Music Library</title>
+
+    <!-- If IE use the latest rendering engine -->
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+    <!-- Set the page to the width of the device and set the zoon level -->
```
Comment (tadityar) [inline app/templates/base.html]: typo

File: app/templates/base.html [hunk 2]
```diff
@@ -1,15 +1,29 @@
 <!DOCTYPE html>
-<html>
+{% load staticfiles %}
+<html lang="en">
   <head>
     <meta charset="utf-8">
-    <title>IMSLPBAR/not Petrucci Music Library</title>
+
+    <!-- If IE use the latest rendering engine -->
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```
Comment (tadityar) [inline app/templates/base.html]: I think it's ok if we don't support IE hahahahha

File: imslpbar/settings_template [hunk 3]
```diff
@@ -76,7 +76,7 @@ DATABASES = {
         'ENGINE': 'django.db.backends.mysql',
         'NAME': 'imslpbar',
         'USER': 'root',
-        'PASSWORD': '',
+        'PASSWORD': 'root',
```
Comment (tadityar) [inline imslpbar/settings_template]: this change should not be here, if you need to change it do it locally. Settings template should be the "general" template that everyone can copy paste to create their `settings.py`

File: manage.py [hunk 4]
```diff
@@ -3,7 +3,7 @@
 import sys
 
 if __name__ == "__main__":
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "imslpbar.settings")
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "imslpbar.settings_template")
```
Comment (tadityar) [inline manage.py]: and it was not changed to settings_template. You can read the README for this.

File: app/static/css/app.css [hunk 5]
```diff
@@ -0,0 +1,8 @@
+body {
+    padding-left: 15px;
+    
+}
+
+h1 a {
+    color: #66FFCC;
+}
```
Comment (tadityar) [inline app/static/css/app.css]: newline at end of file please

File: app/templates/base.html [hunk 6]
```diff
@@ -1,15 +1,29 @@
...
+
     <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.3/umd/popper.min.js" integrity="sha384-vFJXuSJphROIrBnz7yo7oB41mKfc8JzQZiCq4NCceLEaO4IHwicKwpJf9c9IpFgh" crossorigin="anonymous"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/js/bootstrap.min.js" integrity="sha384-alpBpkh1PFOepccYVYDB4do5UnbKysX5WZXm3XxPqe5iKTfUKjNkCk9SaVuEZflJ" crossorigin="anonymous"></script>
-    {% load staticfiles %}
+
+    
   </head>
   <body class="content-fluid">
+    <div class="container">
```
Comment (tadityar) [inline app/templates/base.html]: looking at imslp.org. The content was "centered" but the navbar still spans until the end of the page. I think in this case the use of class `container` should be elsewhere not here.

Comment (monica-nathalia) [reply app/templates/base.html]: sip bos

File: imslpbar/settings_template [hunk 7]
```diff
@@ -1,124 +0,0 @@
...
-
-USE_I18N = True
-
-USE_L10N = True
-
-USE_TZ = True
-
-
-# Static files (CSS, JavaScript, Images)
-# https://docs.djangoproject.com/en/1.11/howto/static-files/
```
Comment (tadityar) [inline imslpbar/settings_template]: don't delete the file :(

File: app/templates/base.html (hunk 6 above)
Comment (tadityar) [reply app/templates/base.html]: so delete this? ehehe

File: app/templates/base.html (hunk 1 above)
Comment (tadityar) [reply app/templates/base.html]: change to zoom

File: app/templates/base.html (hunk 2 above)
Comment (tadityar) [reply app/templates/base.html]: remove

File: app/static/css/app.css (hunk 5 above)
Comment (tadityar) [reply app/static/css/app.css]: one blank line at every end of file

**Pull request merged**